### PR TITLE
Add eng and placeholders for reset metadata notice

### PR DESCRIPTION
**Pull request by kadisonm**
For [this](https://github.com/sboesen/remotely-sync/pull/127).

Thank you to @bytemain for the cn and tw translations.

### CODE REVIEW

**Comment by bytemain** [comment]
I can help fill these chinese placeholder out, can I just make comments?

File: zh_cn.json [hunk 1]
```diff
@@ -262,7 +262,9 @@
   "settings_outputbasepathvaultid_button": "输出",
   "settings_reset_sync_metadata": "重置同步元数据",
   "settings_reset_sync_metadata_desc": "用于删除远程存储的元数据文件。将忽略已删除的文件，因此请确保首先在所有设备上同步。",
-  "settings_reset_sync_metadata_notice": "远程元数据文件已删除。",
+  "settings_reset_sync_metadata_notice_start": "Deleting, please wait... (1/2)",
```
Comment (bytemain) [inline zh_cn.json]: 删除中，请等待... (1/2)

File: zh_cn.json [hunk 2]
```diff
@@ -262,7 +262,9 @@
...
+  "settings_reset_sync_metadata_notice_end": "远程元数据文件已删除。(2/2)",
+  "settings_reset_sync_metadata_notice_error": "Already deleting, please wait.",
```
Comment (bytemain) [inline zh_cn.json]: 已开始删除远程元数据文件，请耐心等待。

**Comment by kadisonm** [comment]
Thanks, I appreciate that! By the way, I just added another one for if they try to reset while it is already resetting.

Edit: You got it thanks!

File: zh_tw.json [hunk 3]
```diff
@@ -262,7 +262,9 @@
   "settings_outputbasepathvaultid_button": "輸出",
   "settings_reset_sync_metadata": "重置同步中繼資料",
   "settings_reset_sync_metadata_desc": "用於删除遠端儲存的中繼資料檔案。將忽略已删除的檔案，囙此請確保首先在所有設備上同步。",
-  "settings_reset_sync_metadata_notice": "遠端中繼資料檔案已刪除。",
+  "settings_reset_sync_metadata_notice_start": "Deleting, please wait... (1/2)",
```
Comment (bytemain) [inline zh_tw.json]: 刪除中，請等待... (1/2)

File: zh_tw.json [hunk 4]
```diff
@@ -262,7 +262,9 @@
...
+  "settings_reset_sync_metadata_notice_end": "遠端中繼資料檔案已刪除。(2/2)",
+  "settings_reset_sync_metadata_notice_error": "Already deleting, please wait.",
```
Comment (bytemain) [inline zh_tw.json:267]: 已開始刪除遠程元數據文件，請耐心等待。

Comment (bytemain) [inline zh_tw.json]: 已開始刪除遠端中繼資料檔案，請耐心等待。

**Comment by kadisonm** [comment]
I pushed them to the branch. Thanks a lot!

File: zh_tw.json [hunk 5]
```diff
@@ -262,7 +262,9 @@
   "settings_outputbasepathvaultid_button": "輸出",
   "settings_reset_sync_metadata": "重置同步中繼資料",
   "settings_reset_sync_metadata_desc": "用於删除遠端儲存的中繼資料檔案。將忽略已删除的檔案，囙此請確保首先在所有設備上同步。",
-  "settings_reset_sync_metadata_notice": "遠端中繼資料檔案已刪除。",
+  "settings_reset_sync_metadata_notice_start": "刪除中，請等待... (1/2)",
+  "settings_reset_sync_metadata_notice_end": "遠端中繼資料檔案已刪除。(2/2)",
+  "settings_reset_sync_metadata_notice_error": "已開始刪除遠程元數據文件，請耐心等待。",
```
Comment (bytemain) [inline zh_tw.json:267]: 已開始刪除遠端中繼資料檔案，請耐心等待。

@kadisonm oh, I made a update for this sentence, cause the words we use everyday are not the same

**Review by bytemain — CHANGES_REQUESTED** [review]

**Comment by sboesen** [comment]
Thank you both for the update & translations!! Merged.

**Pull request merged**